### PR TITLE
Dead bodies after enemies' deaths

### PR DESCRIPTION
**Pull request by Mixss**
# Pull request review checklist 📝 ✅

This is a checklist that helps conduct a pull request review properly. These points are required to merge the PR

- [x] There are no merge conflicts

- [x] The pull request is named accordingly (description of the functionality - it
   will be visible in the changelog later)
   
- [x] Every task requirement (from Jira) is done

- [x] The app runs, it boots up properly (e.g. no syntax errors)

- [x] The proposed functionality is working properly (you need to pull this branch
   and check for yourself)

- [ ] Each function, class or functionality is properly documented
   (short description of what it does, parameters or return values if they need
   any explanation)

### CODE REVIEW

File: dnd-bot/dnd_bot/logic/game/handler_attack.py [hunk 1]
```diff
@@ -53,9 +54,12 @@ async def handle_attack(source: Creature, target: Creature, token) -> (bool, str
 
         target.hp -= (base_damage + weapon_damage)
 
+        # dead of the creature
```
Comment (Silentsky0) [inline dnd-bot/dnd_bot/logic/game/handler_attack.py:57]:
```suggestion
        # death of the creature
```

File: dnd-bot/dnd_bot/logic/prototype/entities/misc/dead_body.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from dnd_bot.logic.prototype.entity import Entity
+
+
+class DeadBody(Entity):
```
Comment (Silentsky0) [inline dnd-bot/dnd_bot/logic/prototype/entities/misc/dead_body.py:4]: zmieńmy nazwę klasy na `Corpse`

File: dnd-bot/dnd_bot/logic/prototype/entities/misc/dead_body.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+    entity_name = "Dead body"
```
Comment (Silentsky0) [inline dnd-bot/dnd_bot/logic/prototype/entities/misc/dead_body.py:6]: tu też

File: dnd-bot/dnd_bot/logic/prototype/entities/misc/corpse.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+class Corpse(Entity):
+
+    entity_name = "Corpse"
+    sprite_path = "dnd_bot/assets/gfx/entities/hole.png"
+
+    def __init__(self, x, y, game_token, sprite_path=None):
+        if sprite_path is not None:
+            Corpse.sprite_path = sprite_path
+        super().__init__(x=x, y=y, game_token=game_token, name=Corpse.entity_name, sprite=Corpse.sprite_path, fragile=True)
```
[the remark this answers is not in the document]
Comment (Mixss) [reply dnd-bot/dnd_bot/logic/prototype/entities/misc/corpse.py:12]: konstruktor będzie odpalany zawsze, po prostu mamy możliwość podania innej tekstury niż podstawowa

**Review by Silentsky0 — APPROVED** [review]

**Pull request merged**